src/MockBot.Api/Services/ChatService.cs

```diff
@@ -6,28 +6,37 @@ namespace MockBot.Api.Services
 {
     public class ChatService : IChatService
     {
-        private readonly IDictionary<Guid, Chat> _chats;
+        private static IDictionary<Guid, Chat>? Chats { get; set; }
```

Comment:
It might be worth letting IOC handle this.  Rather than having this as static - simply mark this implementation as a singleton and you would get the same behaviour (I believe) without uncertainty around object creation I'm seeing in the code below.

Or you could implement a static constructor to assign the static value here [static constructor](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/static-constructors)

...but making this a singleton and making Chats a plain ol' instance variable might be easier to get the same outcome.

Reply:
Actually - I can see the ChatService is a singleton - I'm wondering why the chats dictionary needs to be static?  Can I help in any way here?

Reply:
I've made a class singleton by adding it to `Program.cs` but how would I do it for a field?